recipes/leptonica/all/conanfile.py

```diff
@@ -72,15 +72,15 @@ def requirements(self):
         if self.options.with_jpeg == "libjpeg":
             self.requires("libjpeg/9e")
         elif self.options.with_jpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/3.0.1")
+            self.requires("libjpeg-turbo/3.0.2")
         elif self.options.with_jpeg == "mozjpeg":
             self.requires("mozjpeg/4.1.5")
         if self.options.with_png:
-            self.requires("libpng/1.6.40")
+            self.requires("libpng/[>=1.6 <2]")
         if self.options.with_tiff:
             self.requires("libtiff/4.6.0")
         if self.options.with_openjpeg:
-            self.requires("openjpeg/2.5.0")
+            self.requires("openjpeg/2.5.2")
```

Comment:
This'd create conflicts with every other recipe that depends on openjpeg, as those are stuck on .0 for now

Reply:
opencv has moved to 2.5.2, leptonica is only used by tesseract which is only used by opencv so we might as well update this no ?

Reply:
@RubenRBS, do you need more information ?

Reply:
No, sorry for the delay, will try to push this one forward :)